src/boot/apollo.ts

```diff
@@ -0,0 +1,32 @@
+import { ApolloClient /*, createHttpLink */ } from '@apollo/client/core';
+import { ApolloClients } from '@vue/apollo-composable';
+import { boot } from 'quasar/wrappers';
+import { getClientOptions } from 'src/apollo';
+
+export default boot(
+  /* async */ ({ app }) => {
+    // Default client.
+    const options = /* await */ getClientOptions(/* {app, router ...} */);
+    const apolloClient = new ApolloClient(options);
+
+    // // Additional client `clientA`
+    // const optionsA = { ...options }
```

Comment:
can we clean up this code?

Reply:
This code provided by quasar will be useful in the future when we have more than one GraphQL source in the portal.